src/pages/layer-2.js

```diff
@@ -0,0 +1,614 @@
+      ],
+      alt: "test",
+    },
+    {
+      image: privacyGlyph,
+      title: "Security",
+      preview: "Layer 2 is cashing a cheque, layer 1 is going to court",
+      body: [
+        "We need need scaling solutions that inherit Ethereums security. Layer 2 networks settle their transactions on the Ethereum mainnet, allowing users who use them to benefit from the security of the Ethereum network.",
+        "Layer 2's take advantage of the consensus mechanism of its parent Layer 1, in this case Ethereum, instead of providing their own consensus mechanism.",
+      ],
+      alt: "test",
+    },
+  ]
+
+  const layer2DataCombined = [...layer2Data.optimistic, ...layer2Data.zk]
+
+  return (
+    <GappedPage>
+      <PageMetadata
+        title={"Layer 2"}
+        description={"Introduction page to layer 2"}
+      />
+
+      <HeroContainer>
+        <Hero content={heroContent} isReverse />
+      </HeroContainer>
+
+      <Content>
+        <h2>What is Layer 1?</h2>
+        <p>
+          Layer 1 blockchains, like Ethereum and Bitcoin, are the underlying
+          foundation that layer 2 projects build on top of. Some examples of
+          layer 2 projects being build on top of these networks are zk-rollups
+          and optimistic-rollups on Ethereum, and the Lighting Network on top of
+          Bitcoin.
+        </p>
+        <p>
+          Ethereum as the layer 1 includes the blockchain itself and the history
+          of transaction data, a network of miners (soon to be validators after{" "}
+          <Link to="/upgrades/merge/">The Merge</Link>) proposing blocks, a
+          network of node operators to secure and validate the network, and the
+          consensus mechanism for the network. Ethereum also function as a data
+          availability layer for Layer 2's, and if there are any disputes on
+          previous transactions data is provided from Ethereum for these
+          disputes.
+        </p>
+        <p>
+          While Ethereum has been a breakthrough technology in smart contract
+          blockchains, it faces the same trilemma between scaling,
+          decentralization, and security as any other blockchain. Ethereum has
+          done a great job addressing decentralization, and security, with plans
+          to address scalability with sharding in the future. While there are
+          roadmap plans to improve the scalability of Ethereum, there is still a
+          need for further scaling to round out the trilemma for Ethereum. This
+          is where Layer 2 technology is being developed to step in and bring
+          further scaling to Ethereum. As time has progressed, Ethereum as a
+          layer 1 is a settlement layer for layer 2's, and not the layer for end
+          users.
+        </p>
+      </Content>
+
+      <Content>
+        <h2>Why do we need layer 2?</h2>
+        <p>
+          The three desirable properties of a blockchain are that it is
```

Comment:
This is a good introduction of the scalability trilemma. I don't think mentioning it before this (as we do above) makes sense.

Reply:
<img width="739" alt="image" src="https://user-images.githubusercontent.com/54227730/159772214-9245f11f-8dae-4ca6-aec9-b4ed6ca83d86.png">

```suggestion
          The three desirable properties of a blockchain are that it is{" "}
```